app/component/summary/summary-row.cjsx

```diff
@@ -43,6 +42,9 @@ class SummaryRow extends React.Component
       # Use either vehicle number or walking distance as text
       if leg.transitLeg and leg.mode.toLowerCase() == 'subway'
         text = " M"
+      else if leg.mode == 'WAITING'
+        # Use waiting time in minutes
+        text = " #{Math.floor(leg.duration / 60000)}min"
```

Comment:
timeUtils doesn't have a suitable function, we use rounded minutes elsewhere, so switched from floor to that.